cypress/integration/version/downstream_tasks.ts

```diff
@@ -17,7 +17,7 @@ describe("Downstream Tasks Tab", () => {
     // wait is used to prevent clicking on a detached project-accordion element
     // https://github.com/cypress-io/cypress/issues/7306
     cy.wait(0);
-    cy.dataCy("project-accordion").first().click();
+    cy.dataCy("accordion-toggle").first().click();
```

Comment:
Changed this dataCy since the changes to Accordion adds an additional div to keep the title and caret in the same flex-row in the case `allowToggleFromTitle` is set to false.